src/playercards/CardSearch.ttslua

```diff
@@ -87,50 +105,29 @@ function search()
         return
     end
 
-    local cardList = allCardsBag.call("getCardsByName", { name = INPUT_BOX, exact = searchExact })
+    -- search all objects in bag
+    local cardList = allCardsBag.call("getCardsByName", { name = inputParameters.value, exact = searchExact })
     if cardList == nil or #cardList == 0 then
         printToAll("No match found.", "Red")
         return
     end
 
-    -- search all objects in bag
-    local spawnCount = 0
-    for i, card in ipairs(cardList) do
-      local pos = self.positionToWorld(Vector(0, 0.5 + spawnCount * 0.15, -0.225))
-      local rot = self.getRotation()
-      spawnObjectData({
-        data = card.data,
-        position = pos,
-        rotation = rot,
-      })
-      if not spawnAll then
-        return
-      end
-    end
-end
-
-function input_func(_, _, input, stillEditing)
-    if not stillEditing then INPUT_BOX = input end
-end
-
--- toggle spawn mode
-function spawnMode()
-    spawnAll = not spawnAll
-
-    if spawnAll then
-        self.editButton({ index = 1, label = "Mode: Spawn all matching cards " })
-    else
-        self.editButton({ index = 1, label = "Mode: Spawn first matching card" })
-    end
-end
-
--- toggle search mode
-function searchMode()
-    searchExact = not searchExact
-
-    if searchExact then
-        self.editButton({ index = 2, label = "Mode: Name matches search term" })
-    else
-        self.editButton({ index = 2, label = "Mode: Name contains search term" })
+    -- sort table by name (reverse for multiple results, because bottom card spawns first)
```

Comment:
Feel free to leave it for later, but this could include and new the new PlayerCardSpawner